mergin/client.py

```diff
@@ -398,6 +398,28 @@ def pull_project(self, directory):
         pull_project_wait(job)
         return pull_project_finalize(job)
 
+    def clone_project(self, full_project_name, namespace, name):
```

Comment:
```suggestion
    def clone_project(self, source_project_path, cloned_project_name, cloned_project_namespace=None):
```

few suggestions:
- other methods use `project_path` as the "full" project name, here we need to distinguish also the source and destination so I suggest `source_project_path`
- `name` -> `cloned_project_name`
- `namespace` - this should be optional argument (have default value = None) and use the current user's username if not specified.